frontends/web/src/components/status/status.tsx

```diff
@@ -33,91 +28,63 @@ interface StatusProps {
     className?: string;
 }
 
-type Props = StatusProps;
-
-export default class Status extends Component<Props, State> {
-    public readonly state: State = {
-        show: true,
-    };
-
-    public componentDidMount() {
-        this.checkConfig();
-    }
-
-    public componentDidUpdate(prevProps) {
-        if (this.props.dismissable !== prevProps.dismissable) {
-            this.checkConfig();
+const Status: FunctionComponent<Props> = ({
+    children,
+    className,
+    dismissable,
+    hidden,
+    type = 'warning',
+}) => {
+    const { config, setConfig } = useConfig();
+    
+    let dismissed = false;
+    if (dismissable) {
+        if (!config) {
+            // Wait until config is loaded to prevent rendering a dismissed Status
+            return null;
         }
+        dismissed = !!config.frontend[dismissable]
```

Comment:
It doesn't need `!!`, still it is a way to convert a `undefined | boolean` to a `boolean` using a truthy conversion. 
I like it because the later `if (hidden || dismissed) {` is between booleans, and not more complex types.
WDYT?